packages/engine/src/parseErrors.ts

```diff
@@ -1,39 +1,66 @@
 import tsErrorMessages from './tsErrorMessages.json';
 
+type TsErrorMessageDb = Record<string, { code: number }>;
```

Comment:
It's not really a db 😅 . I don't know, I'd personally name it `TsErrorMessageRecord` since the type implies it's a record but ok it's fine either way.